ultravox/training/config_base.py

```diff
@@ -80,6 +107,16 @@ class TrainConfig:
     loss_config: Optional[ultravox_config.LossConfig] = None
 
     def __post_init__(self):
+        if self.data_dicts:
+            self.data_dicts = [
+                DataDictConfig(**data_dict) for data_dict in self.data_dicts
+            ]
+            if self.data_sets:
+                self.data_sets.extend(self.data_dicts)
+            else:
+                self.data_sets = self.data_dicts
```

Comment:
Why do you need the `if` condition? `self.data_sets` can never be None.

Reply:
I think you are right.